its/ruling/src/test/resources/jdk6/squid-S5411.json

```diff
@@ -0,0 +1,12 @@
+{
+'jdk6:java/awt/Component.java':[
+5968,
+],
+'jdk6:java/awt/GraphicsEnvironment.java':[
+114,
```

Comment:
we should avoid having duplicate issue, you can use `BaseTreeVisitor` to cut the visit